src/main/webapp/destinationCreation.js

```diff
@@ -22,6 +22,7 @@ function addScriptToHead() { // eslint-disable-line
 }
 
 function searchForPlace() { // eslint-disable-line
+  document.getElementsByClassName('search-results').innerHtml = '';
```

Comment:
I might make 'search-results' a const, since this is used more than once in this file.

Reply:
Yea you are right! thanks